invenio_opendata/testsuite/data/cms/cms-eventdisplay-files-Run2011A.xml

```diff
@@ -47,9 +47,9 @@
       <subfield code="e">CMS</subfield>
     </datafield>
     <datafield tag="772" ind1=" " ind2=" ">
-      <subfield code="a">FT_53_LV5_AN1</subfield>
-      <subfield code="o"></subfield>
-      <subfield code="w">1</subfield>
+      <subfield code="a">/BTag/Run2011A-12Oct2013-v1/AOD</subfield>
+      <subfield code="o">10.7483/OPENDATA.CMS.N372.QF6S</subfield>
+      <subfield code="w">15</subfield>
```

Comment:
That's weird, it's already in the records of last year's event display files...
I would only use it for DOIs and cover other cases with $u and the URL